frontend/src/components/lobby/create-lobby/Lobby.jsx

```diff
@@ -56,8 +52,16 @@ export default function App() {
 
     }, [socket]);
 
+    useEffect(() => {
+        if (socket && IsOwner(localUsername)) {
+            socket.emit('lobby data changed', localLobby, lobbyData);
+        }
+    });
+
     const handleBeforeUnload = useCallback((event) => {
         event.preventDefault();
+        console.log('lobbyData', JSON.stringify(lobbyData))
```

Comment:
Ezt azt hiszem én írtam, de ide fölösleges a JSON.stringify. Sima objectként is ki tudná írni méghozzá színesen